mkdocs.yml

```diff
@@ -30,9 +30,9 @@ nav:
         - How to use the query tool: "query_tool.md"
         - Querying the API directly: "api.md"
     - Setting up your own Neurobagel node:
-        - Getting started: "getting_started.md"
-        - Configuring a node: "config.md"
-        - Maintaining a node: "maintaining.md"
+        - Getting started: "guide/getting_started.md"
```

Comment:
I think moving the deployment pages into a dedicated directory/path makes sense, but the linked issue https://github.com/neurobagel/documentation/issues/99, as far as I recall, concerned moving all Neurobagel _tool usage_-related instructions (i.e., all pages categorized under the docs tab "User Guide") into its own directory, with a path like `user-guide`. Basically, making the current visible structure of the docs reflected in the path structure.

e.g., https://neurobagel.org/overview/ would become https://neurobagel.org/user-guide/overview/, https://neurobagel.org/data_prep/ becomes https://neurobagel.org/user-guide/data_prep/, etc.

This way, we have more intuitive & fewer top-level docs paths (and in the repo), and can direct a user directly to  https://neurobagel.org/user-guide/overview/ if they want to get started using our tools. I think we could still have a `deploy` subpath for just the deployment guide, but our "user guide" captures not just deployment instructions, but also instructions for the query tool, annotation tool, etc.

This obviously would require changing more page links, so if this seems too big for this PR, maybe we could just rename the current `guide` path to `deploy`, and then leave #99 open to eventually move everything under `user-guide`?

Let me know if that makes sense!

Reply:
Aha, good call. In that case, let's do it in its own PR - then we can also separately check that we don't mess up relative links. I'll roll the change back here.